src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObj = {};
+
+  const splittedStr = sourceString.split(';');
+
+  splittedStr.map((item) => {
```

Comment:
method map is supposed to be used when you need to return a new array, but in this you are not return anything 
so, make sense to use method forEach here